homeassistant/components/comelit/sensor.py

```diff
@@ -37,22 +72,57 @@ async def async_setup_entry(
 ) -> None:
     """Set up Comelit sensors."""
 
+    if config_entry.data.get(CONF_TYPE, BRIDGE) == BRIDGE:
+        await async_setup_bridge_entry(hass, config_entry, async_add_entities)
+    else:
+        await async_setup_vedo_entry(hass, config_entry, async_add_entities)
```

Comment:
Both functions are quite small and similar, I think we can make this smaller.

Maybe the list type can be `list[SensorEntity]`